### DISCUSSION TITLE
Can I access the wsl file system from windows?

### DISCUSSION THREAD

**Discussion by sparsh-b** [microsoft/WSL#8021] [Q&A]
Is there a way to view the files in /home/<wsl-LinuxDistro_username> in windows GUI?

**Comment by elsaco** [comment]
Yes. Try `\\wsl.localhost\Ubuntu\home\<wsl-username>` in file explorer, for the Ubuntu distro in this case.

Or pwsh:

```
PS C:\> ls \\wsl$\Ubuntu\home

    Directory: \\wsl$\Ubuntu\home

Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d----           1/19/2022  8:25 PM                tux
```

**Reply by sparsh-b** [reply]
I have Ubuntu-18.04 distro. Tried the file explorer solution suggested. It doesn't work. I am getting the following error:
![image](https://user-images.githubusercontent.com/45140884/153393727-cd983d6b-f854-4b40-82f0-aaf907f1b105.png)


pwsh method shows the existance of the user tho:
![image](https://user-images.githubusercontent.com/45140884/153393444-0718857e-0d1a-4a7c-834d-0c4855bb343a.png)

**Reply by sparsh-b** [reply]
I want to open the `/home/<wsl-username>` directory in file explorer

**Reply by kczx3** [reply]
@elsaco I can't get this to work at all.  I just get the same error as @sparsh-b does.  How can I debug what the issue is?

**Reply by kczx3** [reply]
Turns out this was my exact issue https://github.com/microsoft/WSL/issues/4027#issuecomment-496628274
